src/patterns/data/default/vein_and_venule_pattern.tsv

```diff
@@ -37,3 +37,13 @@ UBERON:8600105	lumbar vein	FMA:70889	UBERON:0001638	UBERON:0002355	A vein that i
 UBERON:8420001	ethmoidal vein	FMA:51770	UBERON:0001638	UBERON:0002200	A vein that accompanies an ethmoidal artery and drains the ethmoidal cells.	wikipedia:Ethmoidal_veins|https://www.ncbi.nlm.nih.gov/books/NBK499826/						UBERON:0011241	https://orcid.org/0000-0001-8314-2140	2024-08-28T09:55:00Z
 UBERON:8420002	anterior ethmoidal vein	FMA:51771	UBERON:8420001		An ethmoidal vein that accompanies an anterior ethmoidal artery. The anterior ethmoidal vein normally drains into the inferior root of the superior ophthalmic vein.	https://doi.org/10.1167/iovs.02-0865|https://doi.org/10.1007/978-3-031-40697-3_23				UBERON:0011192			https://orcid.org/0000-0001-8314-2140	2024-08-28T09:55:00Z
 UBERON:8420003	posterior ethmoidal vein	FMA:51772	UBERON:8420001		An ethmoidal vein that accompanies a posterior ethmoidal artery. The posterior ethmoidal vein often drains into a venous network under the orbital roof.	https://doi.org/10.1167/iovs.02-0865|https://doi.org/10.1007/978-3-031-40697-3_23							https://orcid.org/0000-0001-8314-2140	2024-08-28T09:55:00Z
+UBERON:8920010	left marginal vein	FMA:4708	UBERON:0001638	UBERON:0000948	A vein that is a tributary of the great cardiac vein (left coronary vein) and drains blood from the left ventricle of the heart. It courses with the left marginal artery as it travels superiorly along the left cardiac border.	wikipedia:Left_marginal_vein|https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/left-marginal-vein-of-heart/22447				UBERON:0006958		UBERON:0002084	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33

```

Comment:
The elsevier anatomy link does not look like a peer-reviewed publication.  Do you have permission from the copyright holder to use it in Uberon? How persistent is this resource?
> "Copyright © 2025 Elsevier, its licensors, and contributors. All rights are reserved, including those for text and data mining, AI training, and similar technologies."

I would drop the elsevier link. The wikipedia and FMA links should suffice.